source/Introduction/About-SpaceROS.rst

```diff
@@ -1,12 +1,96 @@
 About Space ROS
 ===============
 
-TODO:
+..
+    rst-class:: guidance
 
-The intro to Space ROS starts here.
-We can draw from existing docs to describe topics such as:
+    * What, when, why, how
+    * General approach and philosophy
+    * Supporting the aerospace certification standards: DO-178C, NPR7150-2D
+    * Continuous Qualification and certification artifacts
 
-* What, when, why, how
-* General approach and philosophy
-* Supporting the aerospace certification standards: DO-178C, NPR7150-2D
-* Continuous Qualification and certification artifacts
+There are many types of robotic space systems currently in use, and planned for the future.
+Manipulators, like the Canadarm, that can deploy payloads; humanoid robots, like Robonaut 5, that can perform tasks that require manual dexterity in the hands and fingers; mobility systems, like the VIPER rover, that can traverse planetary surfaces; robotic spacecraft; and robotic landers.
+In the future, robots will be used for all sorts of tasks, like mining and excavation, maintenance and repair, in-space manufacturing, and site preparation.
+
+.. image:: /images/canadarm.jpg
+
+.. rst-class:: image-subtitle
+
+    The Canadarm manipulator
+
+Increase in Software Complexity
+###############################
+
+With these various kinds of applications comes the corresponding increase in the overall complexity of the systems, especially the associated software.
+The amount of software on the robotic systems is increasing at a steep rate.
+Consider the joystick controlled Lunokohd rover of 1970:
+
+.. image:: /images/lunokhod.jpg
+
+.. rst-class:: image-subtitle
+
+    The Lunokhod rover, circa 1970
+
+The amount of code on that rover was certainly minimal when compared to the 2 million+ lines of code required for the Mars rover in 2020.
+Today, writing custom software for each mission is no longer a viable option as this approach typically results in:
+
+* Higher development costs
+* Minimal code reuse
+* Time-consuming and costly qualification to meet required safety standards
+
+The result is that there is an increasing demand for software re-use in space systems.
+
+..
```

Comment:
Is this set of dots spurious? It is treating the following paragraph as a very long comment directive.

Reply:
There was some text that I was trying to keep but comment out. I've removed it.